src/renderer/utils/file-monitor.test.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jest/prefer-expect-assertions */
+import fs from 'fs';
+
+import { FileMonitor } from './file-monitor';
+let a: [Event, string][];
+describe('file-monitor', () => {
+  it('deleting files should be reported', () => {
+    const fileMonitor = new FileMonitor(__dirname);
```

Comment:
Don't use workspace because that will affect source code. Use a dedicated folder or a temp folder (https://stackoverflow.com/a/53757778)